### ISSUE TITLE
Currency label not pointing to input with embedding

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#4785]
To reproduce, create a form `orbeon/a` in Form Builder with a currency field, publish it, and load `http://localhost:8080/orbeon-embedding/?form=a`. Notice how the `for` attribute on the label is missing the embedding prefix.

[+1 from customer](https://3.basecamp.com/3600924/buckets/1966555/messages/3259104091)

**Comment by avernet** [comment]
- [ ] Review [tentative fix](https://github.com/orbeon/orbeon-forms-pe/commit/a7b5a3150bb692acceff044cd9201f69ecbf814e) done on the `2020.1-pe` branch
- [ ] Cherry-pick to `pe` and `ce`
